recipes-kernel/linux/linux-skipjack/0004-usb-gadget-Include-gadget-path-to-resolve-gadgetfs.h.patch

```diff
@@ -0,0 +1,25 @@
+From c73246297beb9507879518533a62b4c0d7eecf41 Mon Sep 17 00:00:00 2001
```

Comment:
It looks like this patch file isn't used. Can you verify this and remove it if that's the case?

Reply:
It's not in linux-skipjack_o.bb so I'll go ahead and remove it